SPM Example/PortalSwift/ViewController.swift

```diff
@@ -2163,6 +2164,7 @@ class ViewController: UIViewController, UITextFieldDelegate {
           swaps.getQuote(args: quoteArgs, forChainId: customChainId) { result in
             guard let transaction = result.data?.transaction else {
               self.logger.error("ViewController.handleSwaps() - ❌ Unable to get quote transaction")
+              self.showStatusView(message: "\(self.failureStatus) Unable to get quote transaction")
```

Comment:
So this is requesting to get the quote to sell 1000 eth? we dont actually submit the transaction is that true?